migrate.go

```diff
@@ -746,6 +817,22 @@ func (m *Migrate) runMigrations(ret <-chan interface{}) error {
 				if err := m.databaseDrv.Run(migr.BufferedBody); err != nil {
 					return err
 				}
+
+				// If there is a post execution function for
+				// this migration, run it now.
+				cb, ok := m.opts.postStepCallbacks[migr.Version]
+				if ok {
+					m.logVerbosePrintf("Running post step "+
+						"callback for %v\n", migr.LogString())
+
+					err := cb(migr, m.databaseDrv)
+					if err != nil {
+						return err
+					}
```

Comment:
I think you should wrap `err` in the return statement with a contextual message using `fmt.Errorf` for better clarity.

```
return fmt.Errorf("failed to execute post migration callback: %w", err)
```